misc/testerina/modules/testerina-runtime/src/main/java/org/ballerinalang/test/runtime/entity/PartialCoverageModifiedSourceFile.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Please check other places as well

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/28767/commits/fffea3e92edd4a82053fde14e89e1d52f4c3b4f5